Sources/Stencil/StringsContext.swift

```diff
@@ -67,7 +70,7 @@ extension StringsFileParser {
     )
     let tables: [[String: Any]] = [[
       "name": tableName,
-      "strings": structuredStrings
+      "levels": structuredStrings
```

Comment:
Shouldn't we keep the `strings` entry too (duplicate of `levels`) for 4.2.1 until we can remove it in 5.0.0, at least f we want to merge that for 4.2.1?